include/aws/mqtt/private/client_impl.h

```diff
@@ -276,7 +318,8 @@ AWS_MQTT_API uint16_t mqtt_create_request(
     void *send_request_ud,
     aws_mqtt_op_complete_fn *on_complete,
     void *on_complete_ud,
-    bool noRetry);
+    bool noRetry,
+    uint64_t packet_size);
```

Comment:
Note: This is a private header, so I *think* it is okay to add an argument to it (and I changed all 6 uses of the function), but I wanted to mention/highlight it just in case.